src/cli/main.go

```diff
@@ -60,16 +60,35 @@ OPTIONS:
 					Usage: `cf detach-as-policy APP_NAME`,
 				},
 			},
+			{
+				Name:     "create-autoscaling-credential",
+				Alias:    "casc",
+				HelpText: "Create custom metric credential for an application",
+				UsageDetails: plugin.Usage{
+					Usage: `cf create-autoscaling-credential APP_NAME [--username USERNAME --password PASSWORD] [--output PATH_TO_FILE]
+
+OPTIONS:
+	--username, -u   Username of the custom metric credential, random username will be set if not specified.
+	--password, -p   Password of the custom metric credential, random password will be set if not specified.
+	--output         Dump the credential to a file in JSON format.
+					`,
+				},
+			},
+			{
+				Name:     "delete-autoscaling-credential",
+				Alias:    "dasc",
+				HelpText: "Delete the custom metric credential of an application",
+				UsageDetails: plugin.Usage{
+					Usage: `cf delete-autoscaling-credential APP_NAME`,
+				},
+			},
 			{
 				Name:     "autoscaling-metrics",
 				Alias:    "asm",
 				HelpText: "Retrieve the metrics of an application",
 				UsageDetails: plugin.Usage{
 					Usage: `cf autoscaling-metrics APP_NAME METRIC_NAME [--start START_TIME] [--end END_TIME] [--asc] [--output PATH_TO_FILE]
 
-METRIC_NAME: 
-	memoryused, memoryutil, responsetime, throughput, cpu.	
-
```

Comment:
An explaination of METRIC_NAME parameter is still required.  Please add  custom metrics